app/routes/tags.rb

```diff
@@ -0,0 +1,13 @@
+module ExercismWeb
+  module Routes
+    class Tags < Core
+      get "/tags/search" do
+        please_login
+
+        tags = Tag.find_by_similarity(params["q"])
+
+        json tags
```

Comment:
Oooh. Damn, I forgot about the API not doing the logged in user thing. I can't remember how we've done that before. Did we have API endpoints that shared a session secret?